.gitignore

```diff
@@ -14,3 +14,7 @@ docs/.cache/
 # Allure
 allure/
 allure-report/
+
+# storage specific
+build/
+services/storage/*
```

Comment:
not sure about line 20, why should it be ignored?

Reply:
I believe, we don't want to push static files to git (built via storage cmake/make)